metabench.cmake

```diff
@@ -377,6 +377,8 @@ file(WRITE ${CHART_HTML_ERB_PATH}
 "        /* configure the x axis */                                          \n"
 "        var xAxis = chart.xAxis || {};                                      \n"
 "        delete(chart.xAxis);                                                \n"
+"        if(typeof xAxis.axisLabel == 'undefined')                           \n"
+"          xAxis.axisLabel = 'Number of Elements'                            \n"
```

Comment:
I think the best we can say is `n`, and then we must explain what `n` is on a per-chart basis.